sportsdataverse/cfb/cfb_pbp.py

```
@@ -3876,29 +3927,14 @@ def __add_player_cols(self, play_df):
-                        ),
-                        pl.col("text").str.extract(
-                            r"(?i)pass intercepted (?:#\d+\s+)?(?-i:([A-Z][\w'.\-]*(?:\s+[A-Z][\w'.\-]*){0,2}))",
-                            1,
-                        ),
-                    ),
+                .when(
+                    (
+                        (pl.col("type.text") == "Interception Return").or_(
+                            pl.col("type.text") == "Interception Return Touchdown",
+                        )
+                    ).and_(pl.col("pass") == True),
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)